src/helpers/accessControl/rules/structureRules.ts

```diff
@@ -46,4 +46,5 @@ export default async function structureRules(user: IUser, can): Promise<any> {
     },
   });
   can([action.read], ressource.statsTerritoires);
+  can([action.read], ressource.conseillers);
```

Comment:
yes ajout d'un commentaire ; les structures peuvent voir tous les candidats ?

Reply:
peut etre meme j' ne sais pas si faut un $and)
 can([action.read,, ressource.conseillers, {
    statut: { $ne: 'RECRUTE },
    structureId: { $ne: user?.entity.oid },
  });

Reply:
@dienamo On a moyen de faire un genre de : 
can([action.read,, ressource.conseillers, {
statut: { $ne: 'RECRUTE },
structureId: { $ne: user?.entity.oid },
});
(je ne sais pas si il faut un $and)

Reply:
à affiner plus tard si possible, là je ne vois pas comment on peut restreindre ici. Les SA doivent toujours voir les candidats finalisés non dispos (sélectionnés ailleurs)